licenses/views.py

```diff
@@ -180,16 +180,22 @@ def view_license(
     request.path, language_code = normalize_path_and_lang(
         request.path, jurisdiction, language_code
     )
-    if is_plain_text:
-        legalcode = get_object_or_404(
-            LegalCode,
-            plain_text_url=request.path,
-        )
-    else:
-        legalcode = get_object_or_404(
-            LegalCode,
-            license_url=request.path,
-        )
+    # Plaintext disabled
+    # if is_plain_text:
+    #     legalcode = get_object_or_404(
+    #         LegalCode,
+    #         plain_text_url=request.path,
+    #     )
+    # else:
+    #     legalcode = get_object_or_404(
+    #         LegalCode,
+    #         license_url=request.path,
+    #     )
```

Comment:
We should consider removing commented-out code, unless it is intended to be added again later.